src/command.py

```diff
@@ -68,8 +77,10 @@ def cursorless_multiple_target_command(
     ):
         """Execute multi-target cursorless command"""
         args = list(filter(lambda x: x is not NotSet, [arg1, arg2, arg3]))
+        spoken_form = " ".join(last_phrase["phrase"])
```

Comment:
Just to check, if I say "disk take air", this won't pick up the initial "disk" right?

Reply:
It will. So we are going to have some discipline when we do the test cases.

Reply:
Hmm. Ok I'm gonna approve this one just so you're not blocked on adding tests to your other PRs but not sure I'm in love with passing through the whole command to every cursorless command. Let's discuss later

Reply:
Yes I think we could improve this. Just unsure of the best solution right now.